src/main/webapp/style.css

```diff
@@ -36,19 +43,20 @@ h4 {
   margin-bottom: 15px;
 }
 
-p { 
-  color: black;  
+p {
+  color: black;
   display: block;
   font-size: 16px;
   font-style: normal;
   font-weight: normal;
   line-height: 25px;
   margin: auto;
   width: 960px;
+  padding: 25px;
```

Comment:
Keep this alphabetized. Also check if it's correct to change this for all pages. It will definitely change spacing on existing pages.